Better Package Style layout (#71)
pyansys/dev-guide: doc/source/packaging/templates.rst

.. _templates:

#########
Templates
#########

Starting a new project from scratch is a tedious task. For this reason, the
`ansys-templates`_ tool was created to simplify the starting process and
make it more dynamic. Any rendered project will be compliant with the latest
PyAnsys coding and API style guidelines.

The ``ansys-templates`` tool
============================

> The `ansys-templates`_ is a command line interface which provides you with a

Reviewer: ```suggestion
The `ansys-templates`_ is a command line interface that provides you with a
```

> collection of templates. This tool allows you to create new PyAnsys projects

Reviewer: ```suggestion
collection of templates. This tool allows you to create new PyAnsys projects.
```

.. _templates:

#########
Templates
#########

Starting a new project from scratch is a tedious task. For this reason, the
`ansys-templates`_ tool was created to simplify the starting process and
make it more dynamic. Any rendered project will be compliant with the latest
PyAnsys coding and API style guidelines.

The ``ansys-templates`` tool
============================

> The `ansys-templates`_ is a command line interface that provides you with a

Reviewer: ```suggestion
The `ansys-templates`_ tool is a command line interface that provides you with a
```

> collection of templates. This tool allows you to create new PyAnsys projects.
> from scratch in a dynamic way by asking you several questions before generating
> the new project.

Reviewer: ```suggestion
collection of templates. When you use this tool to create a new PyAnsys project, your
responses to the several questions that are asked support dynamic project generation.
```
